playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -136,20 +138,67 @@ export default class ParticipantListPage {
     return download;
   }
 
-/**
- * Filters the participant list to search for a specific participant when given their guid
- * @param participantGUID the guid of the specific participant to search for
- */
-public async filterListByParticipantGUID(participantGUID: string): Promise<void> {
-  const customizeViewPanel = this.filters.customizeViewPanel;
-  await customizeViewPanel.open();
-  await customizeViewPanel.selectColumns('Participant Columns', ['Participant ID']);
+  /**
+   * Filters the participant list to search for a specific participant when given their guid
+   * @param participantGUID the guid of the specific participant to search for
+   */
+  public async filterListByParticipantGUID(participantGUID: string): Promise<void> {
+    const customizeViewPanel = this.filters.customizeViewPanel;
+    await customizeViewPanel.open();
+    await customizeViewPanel.selectColumns('Participant Columns', ['Participant ID']);
 
-  const searchPanel = this.filters.searchPanel;
-  await searchPanel.open();
-  await searchPanel.text('Participant ID', {textValue: participantGUID });
-  await searchPanel.search();
-}
+    const searchPanel = this.filters.searchPanel;
+    await searchPanel.open();
+    await searchPanel.text('Participant ID', {textValue: participantGUID });
+    await searchPanel.search();
+  }
+
+  /**
+   * Returns the guid of the most recently created playwright participant
+   * @param isRGPStudy mark as true or false if this is being ran in RGP - parameter is only needed if method is ran in RGP study
+   * @returns the guid of the most recently registered playwright participant
+   */
+  public async getGuidOfMostRecentAutomatedParticipant(isRGPStudy?: boolean): Promise<string> {
+    //Select the columns to be used to help find the most recent automated participant
+    const customizeViewPanel = this.filters.customizeViewPanel;
+    await customizeViewPanel.open();
+    await customizeViewPanel.selectColumns('Participant Columns', ['Participant ID', 'Registration Date', 'First Name']);
+
+    //Only RGP has a default filter with a different First Name field - make sure to deselect it before continuing
+    if (isRGPStudy) {
+      await customizeViewPanel.deselectColumns('Participant Info Columns', ['First Name']);
+    }
+
+    //First filter the participant list to only show participants registered today and that have the name E2E (usual playwright first name)
+    const searchPanel = this.filters.searchPanel;
+    await searchPanel.open();
+
+    await searchPanel.setTodayFor('Registration Date');
```

Comment:
What happens if no participant created on today's date?

Reply:
didn't consider that - I think changing it to look for pts created within the last week or so would be good (since playwright runs everyday every 4 hours)